Gemfile

```diff
@@ -24,7 +24,7 @@ gem 'dough-ruby',
 gem 'jquery-rails'
 gem 'kaminari'
 gem 'letter_opener', group: :development
-gem 'mas-rad_core', '0.0.93'
+gem 'mas-rad_core', github: 'moneyadviceservice/mas-rad_core', branch: 'temp_integration'
```

Comment:
Yes it's a temporary branch that merges the 2 PRs mentioned in the PR description above. I wanted to PR the two mas-rad_core changes independently (for ease of review), but needed them both here. Thus a temporary branch.

Once they're approved I can release the mas-rad_core gem, and squash out the `[REPLACE]` commit above with a proper version based bump commit.